torchgeo/models/resnet.py

```diff
@@ -18,6 +18,92 @@
     _satlas_transforms,
 )
 
+# Landsat TM-toa 7 channels
+bands_7_channels_tm_toa = [
+    'B1',
+    'B2',
+    'B3',
+    'B4',
+    'B5',
+    'B6',
+    'B7',
+]  # https://developers.google.com/earth-engine/datasets/catalog/LANDSAT_LT05_C02_T1_TOA
```

Comment:
Can we move this comment to above the variable (so line 22)? Similar for others. This will allow more lists to be displayed on a single line